picking_dispatch/picking_dispatch.py

```diff
@@ -182,10 +182,50 @@ def assert_start_ok(self):
                     _('Error'),
                     _('This dispatch cannot be processed until %s') % obj.date)
 
-    @api.multi
-    def action_assign_moves(self):
-        for rec in self:
-            rec.move_ids.action_assign()
+    def action_assign_moves(self, cr, uid, ids, context=None):
```

Comment:
I'd like to keep it like that. The new v7 code is a huge performance improvement for the reservation of big dispatches. The new API lines we're giving up are trivial, so I think that's a win.